supporttools/static/supporttools/css/mobile.less

```diff
@@ -109,6 +109,57 @@ body {
     .form-control { display: inline-block; width: 82%; }
 }
 
+
```

Comment:
If you do something like this in the notices.html template in MyUW you can have the notice admin less there:
`    {% block extra_css %}
        <link href="{% static 'css/notice_admin.less' %}" rel="stylesheet" type="text/less">
    {% endblock %}`